universal_transfer.md

```diff
@@ -0,0 +1,112 @@
+# Universal Transfer Method for Open-Attestation Documents
+
+## Status
+
+Draft
+
+## Rationale
+
+The goal of the universal transfer method is to ensure that a standard is available to all clients interacting with OA documents. This allows different clients to be itneropable with one another. The standard is extremely important if we want to provide a client like an [identity wallet](https://github.com/Open-Attestation/identity-wallet) that supports all kinds of OA document.
+
+Important considerations of the transfer method:
+
+- Reference to resource (uri)
+- Encrypted resource (key)
+- Permitted actions (termsOfUse)
+- Safe transfer (encoding)
+- Universal resolver (deep-linking)
+
+## Proposed Solution
+
+Sample QR code:
+
+![Proposed QR](assets/universal_transfer/proposed-qr.png)
+
+```
+https://openattestation.com/?resource=%7B%22uri%22%3A%22https%3A%2F%2Fsomehostedresources.com%2Fdoc%2F7de3bce4-de62-4628-914e-97d41e642582%22%2C%22enc%22%3A%7B%22type%22%3A%22OPEN-ATTESTATION-TYPE-1%22%2C%22key%22%3A%22aa57eb519fd3c63c42c2f2697e8957198b56fc945c4db18b480c07d2e6485a93%22%7D%2C%22permittedAction%22%3A%5B%22VIEW%22%2C%22STORE%22%5D%7D
+```
+
+Decoded Resource (after `?resource=`):
+
+```json
+{
+  "uri": "https://somehostedresources.com/doc/7de3bce4-de62-4628-914e-97d41e642582",
+  "enc": {
+    "type": "OPEN-ATTESTATION-TYPE-1",
+    "key": "aa57eb519fd3c63c42c2f2697e8957198b56fc945c4db18b480c07d2e6485a93"
+  },
+  "permittedAction": ["VIEW", "STORE"],
```

Comment:
not too clear what this is supposed to do - storing permissions within the QR is dangerous since the QR isn't a secure authorisation method

Reply:
This is to register intent of the sharer, w3c vc's `presentation` is a better model. This can be an interim soln